apps/src/aichat/views/AichatView.tsx

```diff
@@ -169,11 +166,14 @@ const AichatView: React.FunctionComponent = () => {
     dispatch(addNotification(getResetModelNotification()));
   }, [dispatch, levelAichatSettings]);
 
-  const dialogControl = useContext(DialogContext);
+  const dialogControl = useDialogControl();
 
   const onClickStartOver = useCallback(() => {
     if (dialogControl) {
-      dialogControl.showDialog(DialogType.StartOver, resetProject);
+      dialogControl.showDialog({
+        type: DialogType.StartOver,
+        handleConfirm: resetProject,
+      });
```

Comment:
The changes to the non-codebridge, non-dialog files are identical - just changing to the new named args and the new wrapper hook to provide the context.